templates/docs/examples/patterns/adv-grid/default.html

```diff
@@ -0,0 +1,61 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Advanced Grid / Default{% endblock %}
+
+{% block standalone_css %}patterns_adv-grid{% endblock %}
+
+{% block content %}
+
+<div class="adv-grid-demo">
+  <div class="p-adv-grid">
+    <div class="p-adv-grid__title">
+      <p>MORE FROM CANONICAL</p>
```

Comment:
This should be some kind of heading element (usually h2 or h3 for such component) using `p-muted-heading` component class name. 

Not ALL CAPS in HTML.